command/src/command.proto

```diff
@@ -412,6 +412,8 @@ message QueryMetricsOptions {
     repeated string backend_ids = 3;
     // query only these metrics
     repeated string metric_names = 4;
+    // query only worker and main process metrics
```

Comment:
only workers or main process? which ones, this is confusing to me

Reply:
sure, I renamed this option to `no-clusters`